### PR TITLE
HDDS-5912. Add description of hdds.datanode.dir in Getting Started

### PR DESCRIPTION
**Pull request by JunLuo**
## What changes were proposed in this pull request?

add description of datanode data dir in Ozone On Premise Installation of Gettting Started

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5912

### CODE REVIEW

File: hadoop-hdds/docs/content/start/OnPrem.md [hunk 1]
```diff
@@ -104,6 +104,17 @@ Here is an  example,
...
+stored. Defaults to dfs.datanode.data.dir if not specified.
+
+Here is an  example,
+{{< highlight xml >}}
+    <property>
+       <name>hdds.datanode.dir</name>
+       <value>/data/disk1/data</value>
+    </property>
+{{< /highlight >}}
+
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/start/OnPrem.md:117]: This section lists properties which are present in the `ozone-site.xml` config file created by `ozone genconf`.  `hdds.datanode.dir` is not such a property, so I think this section should not list it.

```suggestion
```

File: hadoop-hdds/docs/content/start/OnPrem.md [hunk 2]
```diff
@@ -115,6 +126,7 @@ Here is an  example,
 | ozone.scm.client.address       | SCM server name and port     | Used by client-side                                              |
 | ozone.scm.datanode.address     | SCM server name and port     | Used by datanode to talk to SCM                                  |
 | ozone.om.address               | OM server name               | Used by Ozone handler and Ozone file system.                     |
+| hdds.datanode.dir               | file path                | The HDDS data will be stored here.                     |
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/start/OnPrem.md:129]:
```suggestion
| hdds.datanode.dir               | file path                | HDDS datanodes store data in this directory.           |
```

File: hadoop-hdds/docs/content/start/OnPrem.zh.md [hunk 3]
```diff
@@ -87,6 +87,18 @@ ozone genconf <path>
...
+
+示例如下：
+
+{{< highlight xml >}}
+    <property>
+       <name>hdds.datanode.dir</name>
+       <value>/data/disk1/data</value>
+    </property>
+{{< /highlight >}}
+
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/start/OnPrem.zh.md:101]:
```suggestion
```

**Review by adoroszlai — CHANGES_REQUESTED** [review]
Thanks @JunLuo for documenting this property.

**Comment by JunLuo** [comment]
Because the property "hdds.datanode.dir" doesn't contain "REQUIRED" tag, it is not generated in  genconf. But the default value of "hdds.datanode.dir" can be misleading for user, especially new users. So, Is it necessary to add required tag for hdds.datanode.dir?

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @JunLuo for the patch, and sorry for the in merging it.

**Pull request merged**